cmd/instance_pool_update.go

```diff
@@ -32,13 +30,14 @@ type instancePoolUpdateCmd struct {
 	InstancePrefix     string            `cli-usage:"string to prefix managed Compute instances names with"`
 	InstanceType       string            `cli-usage:"managed Compute instances type (format: [FAMILY.]SIZE)"`
 	Labels             map[string]string `cli-flag:"label" cli-usage:"Instance Pool label (format: key=value)"`
+	MinAvailable       int64             `cli-usage:"Minimum number of running Instances"`
 	Name               string            `cli-short:"n" cli-usage:"Instance Pool name"`
 	PrivateNetworks    []string          `cli-flag:"private-network" cli-usage:"managed Compute instances Private Network NAME|ID (can be specified multiple times)"`
-	SSHKey             string            `cli-flag:"ssh-key" cli-usage:"SSH key to deploy on managed Compute instances"`
+	SSHKey             string            `cli-flag:"ssh-key" cli-usage:"SSH key to deploy on managed Compute instances (can be specified multiple times)"`
```

Comment:
missing an s after SSHKey

Reply:
This one I kept, becuase, I was not sure if to make it more than ssh key or only one, usually user update more than ssk keys?

Reply:
Similar to how it was done in https://github.com/exoscale/cli/pull/620, it should be `SSHKeys` and type is `[]string]`. Then instead of setting `updateReq.SSHKey` (which accepts one SSH key), you set `updateReq.SSHKeys`.

Reply:
Actually ignore my previous comment. This kind of change is out of scope of this PR, let's keep it focused instead. So just remove ` (can be specified multiple times)` suffix in the command usage as it is incorrect (you can only specify a single SSH key).

Reply:
ack!